cypress/e2e/awx/resources/workflowJobTemplateSurveys.cy.ts

```diff
@@ -60,7 +61,7 @@ describe('Workflow Job Templates Surveys', function () {
       reusableTemplateSurveyTestSuite.canCreateMultipleSurvey());
   });
 
-  describe.skip('WFJT Surveys: Launch WFJT with Survey Enabled', function () {
+  describe('WFJT Surveys: Launch WFJT with Survey Enabled', function () {
     before(function () {
       cy.createAwxInventory({ organization: (this.globalOrganization as Organization).id }).then(
```

Comment:
@jerabekjiri it migtht be worth create all resources as part of the test, and do not rely on a previous existent resource on the the sever.